### PR TITLE
Fix formspree. Add possibility to send request through plain HTTP POST

### PR DESCRIPTION
**Pull request by vittorio88**
Please let me know if this is satisfactory.

It works for me. It is live here if you want to test.
https://www.lagobello.com/contact/

### CODE REVIEW

**Comment by vittorio88** [comment]
This pull request fixes https://github.com/devcows/hugo-universal-theme/issues/147 .

File: layouts/partials/contact.html [hunk 1]
```diff
@@ -15,7 +15,11 @@ <h3>{{ i18n "contactForm" }}</h3>
 
                 <div id="contact-message"></div>
 
-                <form class="contact-form" method="post" action="https://formspree.io/{{ .Site.Params.email }}">
+                {{ if .Site.Params.contact_form_ajax }}
```
Comment (salim-b) [inline layouts/partials/contact.html]: You could squeeze this into:

```html
<form class="contact-form-{{ with .Site.Params.contact_form_ajax }}ajax{{ else }}post{{ end }}" method="post" action="https://formspree.io/{{ .Site.Params.email }}">
```

Comment (ryanfox1985) [reply layouts/partials/contact.html]: I agree

**Comment by ryanfox1985** [comment]
can you fix the guide style?

**Comment by vittorio88** [comment]
> can you fix the guide style?

What guide?

File: layouts/partials/contact.html (hunk 1 above)
Comment (vittorio88) [reply layouts/partials/contact.html]: Thanks for the tip.

I updated the pull request as per your request.

**Comment by ryanfox1985** [comment]
yes, the eslint output => https://travis-ci.org/devcows/hugo-universal-theme/builds/517661976#L1613-L1617

**Comment by ryanfox1985** [comment]
can you add the flag false to the example file? exampleSite/config.toml
thanks!

File: static/js/front.js [hunk 2]
```diff
@@ -19,12 +19,13 @@ $(function () {
   animations()
   counters()
   demo()
-  contactForm()
+  contactFormAjax()
```
Comment (ryanfox1985) [inline static/js/front.js]: I'm not sure if it's correct, I would avoid to call `contactFrom()` if it's not present the form with class `.contact-form-ajax`

Comment (ryanfox1985) [reply static/js/front.js]: I agree with the rename `contactFormAjax()`

File: static/js/front.js [hunk 3]
```diff
@@ -41,6 +42,10 @@ function contactForm () {
   })
 }
 
+function contactFormPost(){
```
Comment (ryanfox1985) [inline static/js/front.js]: this function is useless

Comment (vittorio88) [reply static/js/front.js]: That it is. Thanks for the observation.

I have submitted an updated pull request without it.

File: static/js/front.js (hunk 2 above)
Comment (vittorio88) [reply static/js/front.js]: Ok. I left the rename.

File: static/js/front.js (hunk 3 above)
Comment (vittorio88) [reply static/js/front.js]: Thanks for the tip. I got rid of it.

File: static/js/front.js [hunk 4]
```diff
@@ -19,12 +19,12 @@ $(function () {
...
   demo()
-  contactForm()
+  contactFormAjax()
 })
 
 // Ajax contact
-function contactForm () {
-  var form = $('.contact-form')
+function contactFormAjax () {
+  var form = $('.contact-form-ajax')
```
Comment (ryanfox1985) [inline static/js/front.js]: just a small thing, can you return if form is null?

**Comment by ryanfox1985** [comment]
This PR resolves this issue #147

And it's complete than this PR https://github.com/devcows/hugo-universal-theme/pull/152

File: static/js/front.js (hunk 4 above)
Comment (vittorio88) [reply static/js/front.js]: Done. Please review 9ae47ed03515d8d1359dcf84d6b32da98e261ce5.

**Comment by webbinfotech** [comment]
Toggling false so that it's not an AJAX submission works for me. However, toggling true does not, even though I have a gold account. I guess I don't really care as long as it works one way or the other, but I thought you should know that I still saw the same error that I always saw even with a gold account supposedly allowing AJAX.

**Comment by ryanfox1985** [comment]
@webbinfotech the contact form it was done with the old API long time ago.

I check the formspree API => https://help.formspree.io/hc/en-us/articles/360013470814-AJAX-Forms

And they suggest to do in this way:

```javascript
$.ajax({
  url: "https://formspree.io/FORM_ID",
  method: "POST",
  data: {message: "hello!"},
  dataType: "json"
});
```

It would be just change this part: https://github.com/devcows/hugo-universal-theme/blob/3a20cfee9b78db6af8e7a02afc2c3936032cbee3/static/js/front.js#L31-L40 are you up to contribute?

**Pull request merged**